### DISCUSSION TITLE
import issue [Feb 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5089] [Discord Threads]
> 🙋‍♂️ **DaveTeamMember**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can u provide a sample csv file for reproducting the issue?  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### I'm sorry I have no idea why discord doesn't let me upload a file 😩  So I've done a simple repo holding it here : https://github.com/jgiretimhaus/nocodb-issue-test  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Ok, I think that I've kinda cracked the issue but this seems to bring another one on a bigger scope  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Before I was using my SuperAdmin/Owner account and couldn't upload for the love of god because of the is_meta error on my previous screenshot  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### I've now created a new account and gave it the Creator perms  
#  
> 🙋‍♂️ **DaveTeamMember**  
####   
![image0](https://cdn.discordapp.com/attachments/1070652483795423252/1070684369234178068/permissions.PNG)  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Logged in with it and it finally worked seamlessly  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### It's really strange and could be breaking some other stuff  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Referencing your own documentation :  
#  
> 🙋‍♂️ **DaveTeamMember**  
####   
![image0](https://cdn.discordapp.com/attachments/1070652483795423252/1070684878728876052/documentation.PNG)  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### I think as  SuperAdmin and supposedly having ***the equivalent permissions of an "Org Level Creator" and "Owner" for all projects within the organisation.***, not being able to have edit/creation rights on projects I manage is kind of a big issue ahaha  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Ok, other part of the issue, I've been retrying to upload on our Cloud App that might overwrite the permissions access I was talking above :  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Trying to import the csv from the main window doesn't work  
![image0](https://cdn.discordapp.com/attachments/1070652483795423252/1070688497863770133/main_window.PNG)  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### But trying to import the same file from the 3 dots on the left side panel and selecting "CSV File" works perfectly  
#  
> 🙋‍♂️ **DaveTeamMember**  
####   
![image0](https://cdn.discordapp.com/attachments/1070652483795423252/1070690367248269392/3_dots.PNG)  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Really strange behavior  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Probably those two ways of importing are not binded to the same function  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### <@&682859083799199814>  
#  
> 🙋‍♂️ **wkw**  
#### ok will check. can u help create a issue on github?  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### I created it : https://github.com/nocodb/nocodb/issues/5019  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### I added the sample .csv and JSON files and videos  
#  
> 🙋‍♂️ **wkw**  
#### thanks. will follow up.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070652483795423252)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
